wiki/Tournaments/RDOC/2020/en.md

```diff
@@ -0,0 +1,131 @@
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2020-07-01/2020-07-24 |
+| Qualifiers | 2020-08-07/2020-08-16 |
+| Round of 32 | 2020-08-22/2020-08-23 |
+| Round of 16 | 2020-08-29/2020-08-30 |
+| Quarterfinals | 2020-09-05/2020-09-06 |
+| Semifinals | 2020-09-12/2020-09-13 |
+| Finals | 2020-09-19/2020-09-20 |
+| Grand Finals | 2020-09-26/2020-09-27 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 12 months of osu!supporter tag |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 month of osu!supporter tag |
+
+![RDOC 2020 winner badge](img/badge.png)
+
+## Organisation
+
+The Royal Danish osu!Cup 2020 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742) |
+| Map selector | ![][flag_DK] [Arche-](https://osu.ppy.sh/users/2143311), ![][flag_BR] [Dada](https://osu.ppy.sh/users/9119507), ![][flag_DK] [Morrighan](https://osu.ppy.sh/users/12042090), ![][flag_DK] [Spork Lover](https://osu.ppy.sh/users/3417469) |
+| Referee | ![][flag_MX] [Baskest](https://osu.ppy.sh/users/3963408), ![][flag_DK] [Captain](https://osu.ppy.sh/users/2563435), ![][flag_DK] [Gladand](https://osu.ppy.sh/users/8996506), ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_US] [moss-](https://osu.ppy.sh/users/9918921), ![][flag_DK] [My Aim Zogs](https://osu.ppy.sh/users/3722715), ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742), ![][flag_DK] [PolleSving](https://osu.ppy.sh/users/13218204) |
+| Streamer | ![][flag_DK] [Gladand](https://osu.ppy.sh/users/8996506), ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_DK] [My Aim Zogs](https://osu.ppy.sh/users/3722715), ![][flag_DK] [Tenaxzity](https://osu.ppy.sh/users/7395529) |
+| Commentator | ![][flag_DK] [ADanishWaffle](https://osu.ppy.sh/users/8117977), ![][flag_MX] [Baskest](https://osu.ppy.sh/users/3963408), ![][flag_DK] [Haikuna](https://osu.ppy.sh/users/3786974), ![][flag_DK] [Peperonni](https://osu.ppy.sh/users/7392020), ![][flag_DK] [PolleSving](https://osu.ppy.sh/users/13218204), ![][flag_DK] [Power Supply](https://osu.ppy.sh/users/7091513), ![][flag_DK] [Tenaxzity](https://osu.ppy.sh/users/7395529), ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_DK] [Zakura](https://osu.ppy.sh/users/3053599), ![][flag_DK] [Zeezus](https://osu.ppy.sh/users/9125328) |
+| Designer | ![][flag_FI] [Micro](https://osu.ppy.sh/users/4801080), ![][flag_BR] [Hana Fubuki](https://osu.ppy.sh/users/4761039) |
+| Wiki editor | ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742) |
+| Donator | ![][flag_DK] [MadX](https://osu.ppy.sh/users/3295048), ![][flag_DK] [Pennek](https://osu.ppy.sh/users/3723742) |
```

Comment:
I usually mention them below the table. `Something around Generous donations from Akebono, Genjitsu, LLEENN, RandyV, and Tama-desu helped fund some of the prizes.`

But not an error, just preference.